static/app/views/replays/details.tsx

```diff
@@ -99,6 +104,26 @@ function ReplayDetails({
     );
   }
 
+  // If the user has navigated to the replay from an event, then we want to
+  // start the video at the time of the event.
+  if (!defined(initialTimeOffset) && defined(eventTimestamp) && startTimestampMs) {
+    // check if our event timestamp is within the range of the replay
+    const eventTimestampMs = new Date(eventTimestamp).getTime();
+    if (eventTimestampMs >= startTimestampMs) {
+      // check if the event timestamp is the correct format
+      if (eventTimestamp.length === 13) {
```

Comment:
checking the length after we've checked `eventTimestampMs >= startTimestampMs` seems weird.